modules/standard/SysBasic.chpl

```diff
@@ -33,64 +59,115 @@ extern type socklen_t = int(32);
 
 // stdin/stdout/sterr
 //extern proc chpl_cstdin():_file;
+pragma "no doc"
 extern proc chpl_cstdout():_file;
+pragma "no doc"
 extern proc chpl_cstderr():_file;
+pragma "no doc"
 extern proc chpl_cnullfile():_file;
```

Comment:
It seems to me that these functions should only be used by certain IO routines.  I did a quick grepmod/greptests - they are only used once each in the modules (in IO.chpl) and chpl_cstdout is used in a single GMP test (test/modules/standard/gmp/studies/gmp-chudnovsky.chpl).  Would it be reasonable to tighten their scope to only where they are needed and replace the one use in the GMP test?